arm/ui/settings/DriveUtils.py

```diff
@@ -27,7 +27,7 @@ def drives_search():
     context = pyudev.Context()
 
     for device in context.list_devices(subsystem='block'):
-        regexoutput = re.search(r'(/dev/sr\d)', device.device_node)
+        regexoutput = re.search(r'(\/dev\/sr\d{1,2})', device.device_node)
```

Comment:
This will allow for up too 100 drives, so I don't imagine we'll need to change it to a number of any length

Reply:
the current code will only allow for sr0 through sr9 and won't detect anything above that, as you noted the update will resolve sr0 through sr99. More than enough. I would like to see a home PC with 99 drives